### DISCUSSION TITLE
[Bug] 下载 禁漫去码tag本 额外多出的页数

### DISCUSSION THREAD

**Discussion by KirisameYuumio** [hect0x7/JMComic-Crawler-Python#318] [Q&A] [answered]
### 注意事项

- [X] 提ISSUE前，请确认你已经在GitHub Issues和Discussions里搜索过类似的问题，并且没找到想要的答案。
- [x] GitHub Actions问题请到置顶的Issue反馈。

### 发生Bug时，你的使用方式？

Python代码

### 你是否在禁漫官方网页或APP，验证过相应的功能是正常的？

我要反馈的bug无需验证/不方便验证

### option配置+源码/命令

从download_failed_list取得的数据
641299  网页显示总页数:23  下载器中出现了 index 24,25
1038767 网页显示总页数:60 下载其中出现了 index 61,62
msg均为'请求重试全部失败异常'

"请求重试全部失败: [https://cdn-msp.jmapinodeudzn.net/media/photos/1038767/ori], ['www.cdnblackmyth.vip', 'www.cdnxxx-proxy.co', 'www.cdnblackmyth.xyz', 'www.jmapiproxyxxx.vip']"

以1038767的index = 62为例 :
download_url = 'https://cdn-msp.jmapinodeudzn.net/media/photos/1038767/ori'
filename = '.net/media/photos/1038767/ori'
save_path = 'BD/きただりょうま/1038767/1\\.net/media/photos/1038767/ori'   
 (其中BD/きただりょうま/1038767/1是我修改的文件夹路径 \\后部分应该是生成的文件名称部分

应该可以在所有tag包含禁漫去码的本上复现

不清楚是不是因为我没有用JCoins购买的原因
如果是因为没有购买 并且要购买后才能显示并下载 24,25     61,62部分的话  会不会报错信息中提示为需要JCoins购买更合适一点
如果不是哪好像就有点子诡异

[在网页上点阅读会提示购买 但是用这个项目能下载到网页显示的全部页数]

**Comment by hect0x7** [comment, marked as answer]
这个下载失败的图本身就是无法访问的，我也不知道为啥禁漫返了这些异常数据。但是禁漫APP也会请求这些失败图，也会同样失败。
所以我也没做专门过滤这些异常数据。你可以自己写个downloader filter自行过滤。

**Comment by KirisameYuumio** [comment]
明白了
